it/src/test/java/org/corfudb/universe/scenario/OneNodePausedIT.java

```diff
@@ -44,29 +45,28 @@ public void oneNodePausedTest() {
             }
 
             testCase.it("Should pause one node and then resume", data -> {
-                CorfuServer server1 = corfuCluster.getServerByIndex(1);
+                CorfuServer server2 = corfuCluster.getServerByIndex(2);
```

Comment:
The test should work for the original servers. Why changing the servers?

Reply:
See comments above.